.github/workflows/nightly.yml

```diff
@@ -68,25 +68,28 @@ jobs:
         with:
           runTestsParameters: >-
             ${{ matrix.run_tests_parameters }}
-      - name: Test Tracing JIT
+      - name: Test OpCache
```

Comment:
I belive testing opcache prior /w JIT is better

Reply:
It's intentionally chosen this way because tracing jit has a higher chance of failing.

Reply:
yes, exactly - the way it is now will fail the CI pipeline earlier, which is definitely not wanted for nightly pipeline
with opcache /wo JIT first, CI will report correctly if the issue is opcache itself or related /w JIT only
makes sense?